### PR TITLE
Simplify StarPattern solution

### PR DESCRIPTION
**Pull request by Reynadi531**
using repeat function instead of using array or other solution. More info https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

- [x] I have checked that this is not a duplicate problem
- [x] My Code is in running state
- [x] I have read the README.md file 
- [x] I have starred the repo (This is optional though :))
### Description of Change

- [ ] My change is a new addtion.
- [x] My change includes fixes of currently existing code.
- [ ] I have added my solution to some existing problem in the same file.

### CODE REVIEW

File: src/problems/starPattern.js [hunk 1]
```diff
@@ -3,22 +3,20 @@ for a given value of n, print a star pattern starting from 1 star each line to n
 */
```
Comment (vyasriday) [inline src/problems/starPattern.js:4]: @Reynadi531 Please do not remove the previous solution. You can add yours below it.

**Review by vyasriday — CHANGES_REQUESTED** [review]

File: src/problems/starPattern.js (hunk 1 above)
Comment (Reynadi531) [reply src/problems/starPattern.js:4]: Ok i'll make a new PR

Comment (Reynadi531) [reply src/problems/starPattern.js:4]: Done on my master commit

**Comment by vyasriday** [comment]
@Reynadi531  Use https://github.com/vyasriday/JS-Tidbits/blob/master/src/problems/stringReversal.js for reference on how to add multiple solutions to the same problem.

**Review by vyasriday — APPROVED** [review]

**Comment by vyasriday** [comment]
Merged. Thanks for contributing :)

**Pull request merged**